### PR TITLE
ToDo: Lifetable function

### PR DESCRIPTION
**Pull request by nevrome**
Implemented the most important changes mentioned here: #14.

Needs feedback and testing.

### CODE REVIEW

**Comment by nevrome** [comment]
Here's the current version of the rendered documentation: 

[mortAAR.pdf](https://github.com/ISAAKiel/mortAAR/files/998644/mortAAR.pdf)

Better to evaluate the formulas.

File: R/analytical_functions.R [hunk 1]
```diff
@@ -8,9 +8,8 @@
 #' \emph{Kokkotidis/Richter 1991}, \emph{Keyfitz et al. 2005}
 #' for the literature we examined. \cr
 #' The function takes an individual data.frame or a list of
-#' data.frames and respectively returns an object of type
-#' mortaar_life_table or mortaar_life_table_list for which
-#' exist specialised summary, print and plot functions.
+#' data.frames and returns an object of mortaar_life_table_list
```
Comment (MartinHinz) [inline R/analytical_functions.R:11]: an object of class mortaar_life_table_list

File: R/analytical_functions.R [hunk 2]
```diff
@@ -32,20 +29,23 @@
...
+#' values from the first age interval onwards.
+#'
+#' Default setup is 1/3 for every age class <= 5
+#' life years, and 1/2 for the others.
+#'
 #' @return
-#' An object of type mortaar_life_table or
-#' mortaar_life_table_list. Each mortaar_life_table contains
-#' the following variables:
+#' An object of type mortaar_life_table_list.
```
Comment (MartinHinz) [inline R/analytical_functions.R:48]: of class mortaar_life_table_list

**Review by MartinHinz — APPROVED** [review]
Only some stylistic comments communicated individually, otherwise approved

**Comment by nmueller18** [comment]
Why was it necessary to remove all Umlaute?

**Comment by nmueller18** [comment]
I have made some changes to the new branch. However, the many red lines do not look very reassuring ... Did I manage to delete the whole branch by accident?

What happens if the user sets "agecor" to FALSE? It seems that then lx and lx+1 are just added?! This should not be possible.

**Comment by nevrome** [comment]
You  managed to delete almost the complete repository in this branch... Could we please revert back to my last commit?

**Comment by nevrome** [comment]
If "agecor" is FALSE the the acf is simply 1/2 for every age class. Or is there a problem with the implementation?

**Comment by nevrome** [comment]
I'm not sure about the umlauts. Maybe somebody is willing to dig into the depths of roxygen2, latex and pandoc to check if the documentation can be reliably rendered with them present. I was not able to do this since my last system update...

**Comment by nmueller18** [comment]
@nevrome: Interesting :$. I hope you can just play back your local version?

agecor: No, the implementation is correct, but the manual should be clearer about that (I know, a TODO for me).

**Comment by nevrome** [comment]
Ok - I will do the revert. How did you delete all this files? I guess that wasn't intentional.

Concerning the agecor value I wrote some comments within the code. Please ask again if still unclear.

**Comment by nmueller18** [comment]
Intentional? Not really. I got an error message by RStudio/github that it was not able to push my changes and suggested to delete the hidden index file manually. That, obviously, was not such a brilliant idea …

**Comment by nevrome** [comment]
Haha - new kind of problems... The RStudio git GUI is just capable to handle very basic stuff. Maybe it's useful to take a look at the git CLI for the future. BTW this revert was an interesting experience. One of the first times, I had to do this. 

Unfortunately your intended changes in e63858456c9306e18bd96e4abb6ed253295ee124 never made it to their destination like this... You'll have to do it again.

**Comment by nevrome** [comment]
@nmueller18  Okay - Daniel merged. Should we delete this branch now?

**Pull request merged**